src/layers/NetzkartePointLayer/NetzkartePointLayer.js

```diff
@@ -0,0 +1,158 @@
+import qs from 'querystring';
+import VectorLayer from 'react-spatial/layers/VectorLayer';
+import OLVectorLayer from 'ol/layer/Vector';
+import OLVectorSource from 'ol/source/Vector';
+import OLGeoJSON from 'ol/format/GeoJSON';
+import { bbox as OLBboxStrategy } from 'ol/loadingstrategy';
+import { Style as OLStyle, Circle as OLCircle, Fill as OLFill } from 'ol/style';
+import CONF from '../../config/appConfig';
+import layerHelper from '../layerHelper';
+
+class NetzkartePointLayer extends VectorLayer {
+  constructor(options = {}) {
+    let name = 'Stationen';
+    let key = 'ch.sbb.stationen';
+
+    if (options.showAirports) {
+      name = 'Flughäfen';
+      key = 'ch.sbb.flughafen';
+    }
+
+    const vectorSource = new OLVectorSource({
+      format: new OLGeoJSON(),
+      ...(options.useBboxStrategy && { strategy: OLBboxStrategy }),
```

Comment:
I don't find this a good practice. Please use a ternary operator or a condition.